test/Sema/objc_attr_requires_module_3.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-build-swift -parse %s -Xfrontend -verify
+// REQUIRES: OS=linux-gnu
```

Comment:
Yes, I was intended to simulate real world use case in Linux.
But, should I use `#if false` without any `REQUIRES`?